api/proto/disperser/v2/disperser_v2.proto

```diff
@@ -70,13 +70,18 @@ message BlobStatusRequest {
 }
 
 // BlobStatusReply is the reply to a BlobStatusRequest.
+// signed_batch and blob_include_info are only set if the blob status is GATHERING_SIGNATURES or DONE.
```

Comment:
typo: "blob_inclusion_info"

I think it's easier for reading to place the comment about specific field right next to the field below.